packages/typography/src/Link/Link.tsx

```diff
@@ -112,8 +110,9 @@ const Link = InferredPolymorphic<BaseLinkProps, 'span'>(
         className={cx(
           anchorClassName,
           overwriteDefaultStyles,
-          bodyTypeScaleStyles[baseFontSize],
           linkStyles,
+          // @ts-expect-error baseFontSize will not be used an index when undefined
```

Comment:
nit: we could have a function above to generate the class name and write out the if statement so we don't need to add an expect error